src/js/media-controller.js

```diff
@@ -659,7 +667,7 @@ class MediaController extends MediaContainer {
 
   get isPip() {
     return (
-      this.media && this.media == this.getRootNode().pictureInPictureElement
+      this.media && this.media == document.pictureInPictureElement
```

Comment:
Just to be sure, did you test this and the fullscreen version in browsers other than Safari, with MC in the shadow dom? I have a suspicion this doesn't or shouldn't work. It's a bit of a leaky shadow dom if elements in the shadow dom can be exposed on the higher level doc like this.

Reply:
what do you think of just checking `!!document.pictureInPictureElement` if PiP is enabled?

as far as I know there's no browsers that allow more than 1 at a time.
https://www.w3.org/TR/picture-in-picture/#one-pip-window

Reply:
Oh interesting, if contains() can pierce the shadow dom, awesome! I guess if it's `open` then it should be able to.